analyzers/tests/SonarAnalyzer.Test/TestCases/MethodParameterUnused.RoslynCfg.cs

```diff
@@ -346,6 +346,19 @@ partial class MethodAssignedToActionFromPartialClass
         public System.Action<int> MyReference = MethodAssignedToActionFromPartialClass.MyMethod5;
     }
 
+    public partial class PartialMethod
+    {
+        partial void FalsePositive(string one, string two);
+    }
+
+    public partial class PartialMethod
+    {
+        partial void FalsePositive(string one, string two) // Noncompliant FP
+        {
+            Console.Write(two);
+        }
+    }
```

Comment:
What about `namespace Issue_8988` ?

Reply:
I added the issue number and a link to the class name. I like to keep the test case next to the exiting partial class from above.